jekyll/_cci2/yarn.md

```diff
@@ -13,7 +13,9 @@ version:
 The packages it installs can be cached.
 This can potentially speed up builds but, more importantly, can reduce errors related to network connectivity.
 
-## Using Yarn in CircleCI
+## Using Yarn in CircleCI 
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 6. You can fix by inserting `{: #using-yarn-in-circleci }` to the next line